docs/data/toolpad/getting-started/tutorial.md

```diff
@@ -1,4 +1,4 @@
-# Getting started
+# Tutorial
```

Comment:
Because:

<img width="274" alt="Screenshot 2022-07-20 at 23 47 48" src="https://user-images.githubusercontent.com/3165635/180088039-e839d4fe-0758-470d-8fdc-d98d3e94a7a8.png">